src/components/app/Overview/Overview.tsx

```diff
@@ -388,6 +388,20 @@ export default function AppOverview({ appMetaInfo, getAppMetaInfoRes, isJobOverv
         }
     }
 
+    const environmentName = (jobPipeline: JobPipeline): string => {
```

Comment:
I can see this method on multiple places. Can we make this common and reuse?

Reply:
done